docs/proposals/20220708-recommendation-framework-extender.md

```diff
@@ -0,0 +1,48 @@
+# Recommendation Framework Extender
+
+## Summary
+
+This document describes the Crane Recommendation Framework Extender.At present, crane has not provided a mechanism for non-invasive function realization. If the user needs to implement an internal recommendation function, the source code of crane must be modified.Such as kube-scheduler, this proposal is to dynamically extend crane's recommendation capabilities via an http request, while keeping the recommendation "core" simple and maintainable.
+
```

Comment:
If the user needs to implement an internal recommendation function, such as the kube-scheduler, the source code of crane must be modified respectively. This proposal is to ...

Reply:
What's your question?